README.md

```diff
@@ -0,0 +1,10 @@
+# Stream-Pi scripts
+
+## Stream-Pi Client installer script for Raspberry Pi
+
+Copy and paste the following command in your terminal.
+
+```sh
+wget -q -O - \
+https://raw.githubusercontent.com/stream-pi/scripts/HEAD/client-install-raspberry-pi.sh | bash
```

Comment:
this should be replaced with 

```sh
curl -sSL https://install.stream-pi.com/client/raspberry-pi | bash
```

curl method looks cleaner, also the URL `https://install.stream-pi.com/client/raspberry-pi` is a redirect to `https://raw.githubusercontent.com/stream-pi/scripts/HEAD/client-install-raspberry-pi.sh`

Reply:
This was my first option as well but I went with `wget` since it comes pre-installed with Raspberry Pi OS as opposed to `curl` w/c needs to be installed iirc.

Reply:
> This was my first option as well but I went with `wget` since it comes pre-installed with Raspberry Pi OS as opposed to `curl` w/c needs to be installed iirc.

that is weird. i tried it on a fresh install of raspberry pi os (lite) and curl was included with it

Reply:
Okay I might be running an old version then. Updating instructions to use `curl` and new download link.